### PR TITLE
Implementing cleanup of idleTimeout connection

### PR DESCRIPTION
**Pull request by Ranxy**
#98

### CODE REVIEW

**Comment by Ranxy** [comment]
The reason for CI failure looks the same as  #97.

**Comment by wey-gu** [comment]
> The reason for CI failure looks the same as #97.

True, we should take care of it, even a retrigger cannot bypass it :(.
@Aiee  @jievince

File: connection_pool.go [hunk 1]
```diff
@@ -238,3 +247,72 @@ func (pool *ConnectionPool) createConnection() (*connection, error) {
...
+func (pool *ConnectionPool) connectionCleaner() {
+	const minInterval = time.Second
+
+	d := pool.conf.IdleTime
+
+	if d < minInterval {
+		d = minInterval
+	}
+	t := time.NewTimer(d)
+
```
Comment (Aiee) [inline connection_pool.go:268]: We don't need to check the idle status that often. Setting the checking interval to 60 seconds seems more appropriate to me.

**Review by Aiee — COMMENTED** [review]
Good job!

**Review by Aiee — APPROVED** [review]
Well done.

**Comment by jamieliu1023** [comment]
@Ranxy Thanks for your contribution to the Nebula Graph community! This is Jamie with Nebula and I'd like to email you the Nebula Contributor certificate and ship you a mug to mark this special moment. Could you please kindly reach me via jamie.liu(at)vesoft.com and let me know where I should ship the mug? Again, thanks for being a part of the Nebula community!

**Pull request merged**